packages/react-components/react-table/src/components/TableSelectionCell/renderTableSelectionCell.tsx

```diff
@@ -13,7 +14,11 @@ export const renderTableSelectionCell_unstable = (state: TableSelectionCellState
       {state.type === 'checkbox' && slots.checkboxIndicator && (
         <slots.checkboxIndicator {...slotProps.checkboxIndicator} />
       )}
-      {state.type === 'radio' && slots.radioIndicator && <slots.radioIndicator {...slotProps.radioIndicator} />}
+      {state.type === 'radio' && slots.radioIndicator && (
+        <RadioGroupProvider value={{ value: state.checked ? state.radioValue : '' }}>
```

Comment:
It's not really possible to use the `Radio` as a standalone control, the only workaround I could figure out was to render `RadioGroupProvider` above every single `Radio` component

Reply:
Issue in https://github.com/microsoft/fluentui/issues/25206